### PR TITLE
OCPBUGS-18247: manifests: don't include recording rules when Console capability is not enabled

### PR DESCRIPTION
**Pull request by vrutkovs**
In order to avoid additional load on Prometheus the recording rules for kube-apiserver dashboard are not included when Console capability is not enablked . These are not used anywhere else, so it should not affect any other components.

### CODE REVIEW

**Comment by vrutkovs** [comment]
/hold

testing if its indeed disabled on single node

**Comment by vrutkovs** [comment]
/hold cancel

Verified via cluster bot that the cluster without Console capability doesn't have recording rules

**Review by dgrisonnet — APPROVED** [review]
/lgtm
/approve

Holding until merging to 4.14 is possible again
/hold

**Pull request merged**
